cmd/tailscale/cli/debug.go

```diff
@@ -850,6 +851,11 @@ func runTS2021(ctx context.Context, args []string) error {
 		logf = log.Printf
 	}
 
+	netMon, err := netmon.New(logger.WithPrefix(logf, "netmon: "))
```

Comment:
Or the static constructor for short lived processes. Either way.

Reply:
I figured this was a good idea because it does actually retry, and if the network is flaking it'd print useful things... but maybe that's overoptimistic 😓

Reply:
Hm, now that I think on this again, I think I want to keep it this way to better replicate how the "real" code works, in the unlikely chance this matters.